packages/documentation-site/patternfly-docs/content/design-guidelines/usage-and-behavior/usage-and-behavior.md

```diff
@@ -3,119 +3,138 @@ id: Usage and behavior
+|------------------------ | -----------|
+| You want to provide a short (no more than 1-2 lines) explanation of new or unfamiliar UI elements as a simple text string that is only shown “on-demand.” |Tooltip|
+| You want to include formatted text and/or interactive elements in your message body. | Hint or popover |
+| You want the information to persist until the user dismisses it. | Hint or popover |
+|You want the information to be announced by a screen reader whenever the user tabs to an element.| Hint, popover, or tooltip <br /><br /> **Note:** By default, popovers are only triggered when the user clicks on an element. Therefore screen readers will not read popover text when tabbing through an interface. If a popover is triggered on hover (optional behavior), it will behave like a tooltip and its content will be announced whenever a keyboard user tabs to the trigger element. |
+|You want to present additional information that might not be necessary or relevant to all users.| Hint or popover <br /><br /> **Note:** Hint can be used to convey information to advanced users (such as “pro-tips”. However, since a hint adds static content to a page, you should consider whether it’s important for this information to be shown at all times.|
 
 ## Progressive disclosure
-Progressive disclosure is the practice of hiding information or providing it only when needed in order to simplify a user interface. PatternFly supports several components that can be used to progressively disclose information on a page. The [accordion](/components/accordion) component allows content to be placed in stackable, expandable containers so that content can be hidden from view to simplify presentation and reduce the need for scrolling. [Expandable sections](/components/expandable-section) allow designers to hide a single block of content or settings behind a show/hide link. [Expandable field groups](/components/forms/form) allow designers to group form elements into expandable containers.
+Progressive disclosure is the practice of showing and hiding information as needed, in order to simplify a user interface. To progressively disclose content, we recommend using the **[accordion](/components/accordion)**, **[expandable sections](/components/expandable-section)**, and **[expandable field groups](/components/forms/form/#field-groups)**.
 
-| Use case | [Accordion](/components/accordion) | [Expandable section](/components/expandable-section) | [Field groups](/components/forms/form)|
-|---- | :----: | :----: | :----: |
-|You want to group a list of actions, links, or other data into expandable groups.|✔|⛔|⛔|
-|You want to hide advanced or seldomly used options within a form.|⛔|✔|✔|
-|You want to give the user the ability to show or hide chunks of information on a long scrolling page.|⛔|✔|⛔
+- Accordians allow content to be placed in stackable, expandable containers so that content can be hidden from view to simplify presentation and reduce the need for scrolling.
```

Comment:
```suggestion
- Accordions allow content to be placed in stackable, expandable containers so that content can be hidden from view to simplify presentation and reduce the need for scrolling.
```